llama_stack/apis/models/models.py

```diff
@@ -62,7 +62,7 @@ class Models(Protocol):
     @webmethod(route="/models", method="GET")
     async def list_models(self) -> ListModelsResponse: ...
 
-    @webmethod(route="/models/{model_id}", method="GET")
+    @webmethod(route="/models/{model_id:path}", method="GET")
```

Comment:
are there other instances (shields, etc.) where we would need to make the same change?

Reply:
right, would be good to apply to all instances actually

Reply:
added to all such instances.